src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -36,14 +28,6 @@ export default () => `
 						<li class="windows-only linux-only"><a href="command:workbench.action.files.openFile">${escape(localize('welcomePage.openFileLinuxPC', "Open file"))}</a></li>
 					</ul>
 				</div>
-				<div class="section deploy" style="display:${showDeploySection() ? 'block' : 'none'}">
-					<h2 class="caption">${escape(localize('welcomePage.deploy', "Deploy"))}</h2>
-					<ul>
-						<li><a href="command:azdata.resource.sql-image.deploy">${escape(localize('welcomePage.deploy-image', "Deploy SQL Server on Docker…"))}</a></li>
-						<li><a href="command:azdata.resource.sql-bdc.deploy">${escape(localize('welcomePage.deploy-bdc', "Deploy SQL Server big data cluster…"))}</a></li>
-						<li><a href="command:azdata.resource.deploy">${escape(localize('welcomePage.MoreOptions', "More…"))}</a></li>
-					</ul>
-				</div>
```

Comment:
Instead of removing this I'd say we should consider just having it in stable too.

Reply:
I'll push to @alanrenmsft to let me know how to handle this, but as is it can't stay.

Reply:
I’ll handle this one in the separate PR